spec/features/bulk_updates/bulk_updates_location_keywords_spec.rb

```diff
@@ -293,12 +293,11 @@
         before do
           visit collection_path(@find_and_replace_ingest_response['concept-id'])
 
-          find('.tab-label', text: 'Additional Information').click
         end
 
         it 'displays the new keyword' do
-          within '.location-keywords-preview' do
-            expect(page).to have_content('OCEAN ATLANTIC OCEAN NORTH ATLANTIC OCEAN BALTIC SEA', normalize_ws: true)
+          within '#metadata-preview' do
+            expect(page).to have_no_content('OCEAN ATLANTIC OCEAN NORTH ATLANTIC OCEAN BALTIC SEA', normalize_ws: true)
```

Comment:
Looks like you missed this one.   Also should be separated out and say have_content